src/stylesheets/elements/_typography.scss

```diff
@@ -45,6 +45,10 @@ h6 {
   margin-top: 1.5em;
 }
 
+h3.usa-heading.margin-top {
+  margin-top: 3em;
+}
```

Comment:
This is a styleguide-only change so should be in `docs/doc_assets/css/styleguide.scss`.